### PR TITLE
Fly.io Deployment Support

### PR DESCRIPTION
**Pull request by manjularajamani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,18 @@ You can use MicroBin:
...
 ...and many other things, why not get creative?
 
+## How to deploy app into Fly.io server
+
+### Step 1:
+Install the [Fly.io CLI](https://fly.io/docs/hands-on/install-flyctl/)
+
+### Step 2:
+```
+git clone https://github.com/manjularajamani/microbin.git
```
Comment (szabodanika) [inline README.md:81]: Is your fork of MicroBin needed for this to work, or can users clone szabodanika/microbin?

File: README.md
Comment (szabodanika) [inline README.md:1]: I think this should rather go in the wiki, I am trying to keep the README as short as possible

File: fly.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+app = "microbin"
+primary_region = "ams"
+kill_signal = "SIGINT"
+kill_timeout = "5s"
+
+[experimental]
+  entrypoint = ["microbin", "--highlightsyntax", "--private", "--qr", "--editable", "--enable-burn-after"]
```
Comment (szabodanika) [inline fly.toml:7]: Can users easily change these parameters?

**Review by szabodanika — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (manjularajamani) [reply README.md:81]: Hi @szabodanika 

User can fork or clone [Szabodanika/microbin](https://github.com/szabodanika/microbin.git)

File: fly.toml (hunk 2 above)
Comment (manjularajamani) [reply fly.toml:7]: Yes, they can able to change the parameters.

File: README.md
Comment (manjularajamani) [reply README.md:1]: I'll remove the deployment section on the README and re-submit a fresh PR.

**Pull request closed without merging**